### PR TITLE
HELPS-121 Add analytics to developer hub

### PR DESCRIPTION
**Pull request by BrettMugglinOlive**

### CODE REVIEW

**Review by epiccoleman — APPROVED** [review]

**Review by flowermm — CHANGES_REQUESTED** [review]
Let's add tracking to "get started", "access the LDK", and "LDK" menu item as well. And maybe the additional resources? Helps us understand if people are viewing documentation or not via this launch page. I realize this is basically asking to track every button - there isn't a good way to understand how people are getting to the docs currently. This is the one time I wish we had Pendo flow (though maybe GA has an equivalent) - to understand if people are clicking more buttons if they don't find what they want on the first time 🤔

File: docs/src/components/menu/v2menu.tsx [hunk 1]
```diff
@@ -9,7 +9,16 @@ interface LinksProps {
...
 const Links: React.FunctionComponent<LinksProps> = (props) => (
   <ul className={props.listClass}>
-    <li className={props.itemClass}>
+    <li
+      className={props.itemClass}
+      onClick={() => {
+        window.gtag('event', 'Link out', {
+          event_category: 'Navigation',
+          event_label: 'Github Pages',
+          value: 0,
```
Comment (flowermm) [inline docs/src/components/menu/v2menu.tsx:18]: any way to get info on what github page it is for value?

Comment (BrettMugglinOlive) [reply docs/src/components/menu/v2menu.tsx:18]: Well it leads to the root page of of our autogenerated Github pages docs [here](https://open-olive.github.io/loop-development-kit/ldk/javascript/). So I thought this was appropriate, but I can change it to something else

Comment (flowermm) [reply docs/src/components/menu/v2menu.tsx:18]: works for me! ha

**Review by flowermm — APPROVED** [review]

**Pull request merged**